code/game/objects/effects/effect_system.dm

```diff
@@ -715,6 +715,9 @@ steam.start() -- spawns the effect
 /obj/effect/effect/foam/fire
 	name = "fire supression foam"
 	icon_state = "mfoam"
+	
+/obj/effect/effect/foam/fire/enhanced
+	lowest_temperature = 0
```

Comment:
I have a strong sneaking suspicion that this entire PR was a smokescreen to sneak 0K extinguishers back in by adding a bunch of unrelated effects

Reply:
You would be wrong, all of this started with the RPD changes.  That said, this is a way to bring back something atmos players want, in a gated 'not every round' way that also promotes departmental interaction. 

 Anyway, unless having `lowest_temperature = 0` breaks pipenets or ZAS in some horrific way, can we move this conversation from 'maintainer requesting changes' to a comment?

Reply:
It's not atomic enough to be mergeable. Creating a PR where you add a bunch of toys and then stick on an extremely controversial PR with them is the definition of what the repo standard of atomicity is about. I shouldn't have to spell that out for you, since you seem conscious of the PR from back in February.

Reply:
I don't see how adding this behind a gate is "extremely controversial." #25837 which added TECs was not controversial at all. If #25809 was preceeded by a way to go cooler than 6K instead of removing the only way at the time to do so, I don't think it would be controversial either.

Reply:
No. You're not "bringing back something atmos players want",  you're breaking atmos and this ain't going to get merged.

Reply:
Then feel free to suggest a suitable bottom temperature for it

Reply:
After some consultation, I have learned that while 0+ε temperatures are handled ok, actually having 0K gas can lead to runtimes. That being the case, but still needing this upgrade to compete with the standard of 'RCD a hole in the floor' I'm gonna suggest (and push to my branch) a temperature of 16K, which should be extremely noncontroversial -- you get a powerful tool for fighting fires but still need to scrub away the gas after it's cooled.